core/spec/models/spree/stock/sorter/default_first_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Spree
+  module Stock
+    module Sorter
+      RSpec.describe DefaultFirst, type: :model do
```

Comment:
This looks weird. Why not just do:

```ruby
describe Spree::Stock::Sorter::DefaultFirst do
end
```

Reply:
Why the need for `type: :model`?

Reply:
@BenMorganIO I agree that it's weird, but this is how specs are written for the other models too (including `type: :model`), so I chose to stick with the convention here.